perfkitbenchmarker/benchmarks/cloudsuite_web_search_benchmark.py

```diff
@@ -59,27 +59,27 @@
              '$FABAN_HOME/lib/fabandriver.jar:$JAVA_HOME/lib/tools.jar:'
              '$FABAN_HOME/search/build/lib/search.jar')
 
-flags.DEFINE_string('client_heap_size', '2g',
+flags.DEFINE_string('websearch_client_heap_size', '2g',
```

Comment:
I'm gonna check again, but the guys at EPFL once told me that they would want the benchmark to have cloudsuite in name, but I think it is ok for parameters not to have cloudsuite in name.
I see that cloudsuite_web_search_client_heap_size is too long. What about faban_client_heap_size and solr_server_heap_size? Or just faban_heap_size and solr_heap_size? That would distinguish those parameters from other benchmarks (although not the ones which also use solr or faban and have the same option), but still leaves cloudsuite_web_search_num_clients too long. This also requires user to know what components benchmark uses instead of just benchmark name, so maybe it is not a good idea.
What do you think about making an abbreviation for flag names only, like cs_websearch, would that be consistent enough?